app/src/main/res/values/strings.xml

```diff
@@ -72,6 +72,8 @@
     <string name="settings_appearance">Appearance</string>
     <string name="setting_theme">Theme</string>
     <string name="setting_key_theme">theme</string>
+    <string name="setting_precise_date">Precise date by default</string>
```

Comment:
What's _unprecise_ about relative timestamps? 

As I understand it the difference here is between `relative` and `absolute` timestamps. So for me it would make more sense to name this option `Show absolute timestamps` instead of `Precise date by default`

Reply:
I completely agree with you, to be honest I named it like that only because the boolean variable that was already there was named `preciseDate` and I just wanted to stay coherent.

If it's fine to everyone it can be changed to be *relative*/*absolute* time instead of *precise*.